shims/common/src/main/java/org/apache/hadoop/hive/thrift/HadoopThriftAuthBridge.java

```diff
@@ -372,6 +389,46 @@ public TTransportFactory createTransportFactory(Map<String, String> saslProps)
     }
 
     /**
+     * Create a TTransportFactory that, upon connection of a client socket,
```

Comment:
Should the comment mention the need for a configured custom authentication class?